Source/Mosa.Kernel.x86/RTC.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Mosa.Runtime.x86;
```

Comment:
Missing license info. This should be automatically being set by Visual Studio.

// Copyright (c) MOSA Project. Licensed under the New BSD License.

Also, why name to RTC. It's the RTC is part of CMOS.

Reply:
For some reason it's not set by VS. Also, I named it to RTC because it makes more sense: the CMOS stores the BIOS data, while the RTC is the actual clock of the CMOS.